api/policy.go

```diff
@@ -190,8 +190,9 @@ func (s *server) repositoryUserUpdatePolicy() (string, error) {
 
 // repositoryPolicy accepts a list of groups and returns the policy to allow ecr access
 // for resources in the same org/group as well as any passed groups
-func repositoryPolicy(groups []string) (string, error) {
+func repositoryPolicy(account string, groups []string) (string, error) {
 	groupConditions := append([]string{"${aws:ResourceTag/spinup:spaceid}"}, groups...)
+	principal := fmt.Sprintf("arn:aws:iam::%s:root", account)
```

Comment:
This works, but I think you can also use just the account, e.g.
`"Principal": { "AWS": "123456789012" }`

Reply:
interesting, i made the change in the console and this was what came out so i copied it :)